content/news/XXXX-XX-XX-dvs-internals-pt2.md

```diff
@@ -0,0 +1,138 @@
+| -------------- | ------------- |
+| 0              | `000`         |
+| 1              | `001`         |
+| 2              | `010`         |
+| 3              | `011`         |
+| 4              | `100`         |
+| 5              | `101`         |
+| 6              | `110`         |
+| 7              | `111`         |
+
+Of course, that is just a toy example, but even if we only had 8 different positions, we need use 3 bits (= 3 cycles) to represent each position.
+In reality, we want to encode *a lot* more different positions, which means the resulting binary number will be longer, meaning we need more bits and therefore cycles per position we want to encode.
+
+The corresponding signal would look like this:
+
+![Simple bitstream with 3 bits per position]({static}/images/news/timecode-3-bit-bitstream.svg)
+
+The first 3 cycles encode position 0, the next 3 cycles encode position 1, and so on.
+Simple, right?
+Unfortunately, this approach has a major problem.
+
+Let's assume you seek to a random position in the track (by picking up the needle and dropping it somewhere) and the
+next 3 bits are `001`.
+Do you know what position or cycle number we are at?
+
+We can look up `001` in the table above and see that these bits encode position 1 (cycles 3-6).
+So after reading these bits we should be at cycle 6.
+And that is *one* possibility, but not the only one.
+
+![Simple bitstream with 3 bits per position and all occurrences of 001 highlighted]({static}/images/news/timecode-3-bit-bitstream-highlight.svg)
+
+So there are multiple possible possible positions and we don't know which one is the correct one.
+We can mitigate this by reading the next 3 cycles, too, but this doubles the amount of cycles we have to read (6 instead of 3 cycles) until we know the position for sure.
+
+DVS systems are supposed to be responsive and minimize the latency (i.e. the delay between moving the vinyl and seeing/hearing the action in the software) to be usable for scratching.
+Therefore, this approach is not sufficient.
+
+### Using Sentinel Values
+
+The problem with the simple base-2 encoding is that may end up in the middle of a number and we don't know where a number begins or ends.
+If you're a programmer, you might say: "We can use sentinel values to solve this problem."
+A [sentinel value](https://en.wikipedia.org/wiki/Sentinel_value) is a special value that is distinct from all other legal values.
+Such a value could be used to indicate "A new position value starts here".
+
+After a seek, we have to keep reading bits until we encounter the sentinel value, then we can start reading the actual position bits.
+In fact, that's exactly what the early [Final Scratch DVS system](https://en.wikipedia.org/wiki/Final_Scratch#Vinyl/CD_time_code) did.
+Each position was prefixed by `0001`.
+Of course, that bit sequence must not be part of a position bit sequence, but that can be solved by using a slightly more complicated encoding.
+
+#### Example: Final Scratch
```

Comment:
Can we drop this paragraph? For me it includes to much details and distract a bit from understanding the whole.